baton/handler.go

```diff
@@ -14,6 +15,8 @@ func Handler() *mux.Router {
 	return m
 }
 
+var i = 0
```

Comment:
Is this temporary for now? Seems like this'd be the kind of thing we'd want a database for.